scss/components/_contact-card.scss

```diff
@@ -98,25 +100,59 @@
 
 .civicase__contact-avatar {
   background: $contact-avatar-background;
-  min-width: 30px;
-  padding: 5px;
+  min-width: $contact-avatar-size;
+  padding: $crm-btn-padding-small-vertical;
+  position: relative;
 }
 
 .civicase__contact-additional__container--avatar {
   background: $contact-avatar-background;
   margin-left: 0;
   margin-top: -10px;
-  min-width: 30px;
-  padding: 5px;
+  min-width: $contact-avatar-size;
+  padding: $crm-btn-padding-small-vertical;
 }
 
 .civicase__contact-avatar--image {
   background: none;
   padding: 0;
+  position: relative;
+  z-index: 1;
 
   img {
     border-radius: 2px;
     height: 25px;
     width: 25px;
   }
 }
+
+.civicase__contact-avatar__full-name {
+  background: $contact-avatar-background;
+  border-radius: $border-radius-base / 2;
+  display: none;
+  height: $contact-avatar-size;
+  left: 0;
+  padding: $crm-btn-padding-small-vertical $padding-base-horizontal;
+  position: absolute;
+  top: 0;
+  width: auto;
+}
+
+.civicase__contact-avatar--image .civicase__contact-avatar__full-name {
+  border-bottom-left-radius: 0;
+  border-top-left-radius: 0;
+}
+
+.civicase__contact-avatar--has-full-name:hover {
+  opacity: 1;
```

Comment:
Why do we need to set it? As we never set opacity to 0 in the first place.

Reply:
@deb1990 the parent element `.civicase__contact-avatar` also has a `.crm_notification-badge` class. I guess this means the contact avatar is a modified notification badge. This last class adds several styles to the element, but also a sets a `0.8` opacity on hover. This override that behaviour only when we want to show the contact's full name since the opacity was not part of the design.

Should we add a note or fix this in a different way?

Reply:
Okay, its fine then, I get it now. Maybe can we add a comment on the code?